packages/amplication-server/src/core/resource/resourceBtm.service.ts

```diff
@@ -195,11 +197,28 @@ export class ResourceBtmService {
     return JSON.stringify(prompt);
   }
 
+  handleProjectServicesCollision(
+    projectServices: { name: string }[],
+    serviceName: string
+  ): string {
+    let validServiceName = serviceName;
+    projectServices.forEach((service) => {
+      const isDuplicated = service.name === serviceName;
+
+      if (isDuplicated) {
+        const suffix = v4().split("-")[0];
```

Comment:
The ticket says the service name in case of collision should be suffixes with "_new", so why are you using a random suffix here?

Reply:
I guess to avoid collision with multiple runs of btm by ai. I like the randomised name more because it also looks more professional and less hacky

Reply:
I am perfectly fine with this. It just needs to be agreed with Paz since it's a product spec/requirement. If he is good with the randomized solution, I agree.

Reply:
I talked with Paz and he confirmed this implementation